core/locale/application/date_time_service.go

```diff
@@ -83,7 +85,7 @@ func (dts *DateTimeService) GetDateTimeFormatter(timeValue time.Time) (*domain.D
 
 func (dts *DateTimeService) loadLocation() (*time.Location, error) {
 	if dts.location == "" {
-		return nil, errors.New("no location configured")
+		return time.UTC, nil
```

Comment:
The whole if block is unneeded, `time.LoadLocation("")` will return UTC anyways.